podman_compose.py

```diff
@@ -1960,7 +1967,9 @@ def _parse_compose_file(self):
         container_names_by_service = {}
         self.services = services
         for service_name, service_desc in services.items():
-            replicas = try_int(service_desc.get("deploy", {}).get("replicas", "1"))
+            spec_replicas = try_int(service_desc.get("deploy", {}).get("replicas", "1"))
+            replicas = spec_replicas if spec_replicas is not None else 1
```

Comment:
Seems like this changes behavior, doesn't it?

Reply:
I'm looking more from local perspective. In broader picture I agree that changing the behavior makes sense.

In such case we should put this in a separate commit at least and not mix with other refactoring work that doesn't change behavior and can be reviewed without considering wider implications.

By the way, wouldn't this achieve the same and be easier to understand?

`replicas = try_int(service_desc.get("deploy", {}).get("replicas"), fallback=1)`